### PR TITLE
Combine Private Fields and Public Fields into one proposal

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +23,10 @@ This list contains only stage 1 proposals and higher that have not yet been with
 |   | [`s` (`dotAll`) flag for regular expressions](https://github.com/mathiasbynens/es-regexp-dotall-flag)       | Mathias Bynens, Brian Terlson      | 3 |
 |   | [`function.sent` metaproperty](https://github.com/allenwb/ESideas/blob/master/Generator%20metaproperty.md)  | Allen Wirfs-Brock                  | 2 |
 |   | [`String.prototype.{trimStart,trimEnd}`](https://github.com/sebmarkbage/ecmascript-string-left-right-trim)  | Sebastian Markbage                 | 2 |
-|   | [Public Class Fields](https://tc39.github.io/proposal-class-public-fields/)                                 | Jeff Morrison                      | 2 |
+|   | [Class Fields](https://github.com/littledan/proposal-class-fields/)                                         | Daniel Ehrenberg                   | 2 |
```
Comment (chicoxyzzy) [inline README.md]: is @jeffmo also champion?

**Comment by ljharb** [comment]
It'd be appreciated if only proposal champions made PRs about proposals.

**Comment by littledan** [comment]
This change is accurate, except you can now link to it as https://github.com/tc39/proposal-class-fields .

@ljharb I'd be fine with reviewing PRs from contributors to update this document; just mention me and I'll do the review.

File: README.md (hunk 1 above)
Comment (littledan) [inline README.md]: Yes

**Comment by littledan** [comment]
So the two changes needed here are:
- Identifying Jeff Morrison as a cochampion
- Fixing the link to tc39

**Comment by chicoxyzzy** [comment]
- [x] Identifying Jeff Morrison as a cochampion
- [x] Fixing the link to tc39

**Pull request merged**